src/config.template.yaml

```diff
@@ -132,8 +132,6 @@ platforms:
     min_seconds_to_record: 3
 
   dummy:
-    # whether to print messages when methods are called
-    verbose: false
```

Comment:
@dgonano That's exactly my point. Having per platform permutation of contract is bad. What `baseplatform` should do is validate and print warning that specific platform may be misconfigured. Or make sure that platform gets `dict` no matter what (but that would be harder to implement, I guess).